doc/source/releases/0.13.0.rst

```diff
@@ -0,0 +1,141 @@
+using the Sphinx IPython extension to ensure code snippets are always up-to-date,
+and a switch to the `PyData Sphinx theme`_.
+
+Improved filtering
+------------------
+
+For complex datasets, it is often necessary to extract subsets of the data from a Neo object tree by filtering
+based on object types, names, and annotations.
+Neo's filtering capabilities have been improved, with the addition of a new :mod:`filters` module.
+
+For example, suppose we have a dataset from an animal performing a behavioural task.
+Our objective is to retain only the trials where the animal performed correctly,
+for one of the four trial types in the experiment, and where the recordings met certain quality criteria.
+Based on an annotated Neo dataset, we can extract the spike trains of interest in a single step,
+without for loops or complex list comprehensions, as follows:
+
+.. code-block:: python
+
+   import neo.core.filters as nf
+
+   # ... load data, `trial` is a Segment object ...
+
+   sua_spiketrains = trial.filter(
+       objects=neo.SpikeTrain,
+       targdict=[
+           {"sua": True},                                # only single-unit activity
+           {"electrode_reject_HFC": False},              # } exclude different types
+           {"electrode_reject_LFC": False},              # } of artefacts
+           {"electrode_reject_IFC": False},              # } or markers of poor-quality
+           {"noise": False},                             # } recordings
+           {"spike_count": nf.FilterGreaterThan(10000)}  # only use neurons with a sufficiently high firing rate
+       ]
+   )
+
+New IO modules
+--------------
+
+Neo now supports MED_ format, with the new :class`MedIO` and :class:`MedRawIO` classes.
+
+
+Bug fixes and improvements in IO modules
+----------------------------------------
+
+Bug fixes and/or improvements have been made to :class:`AsciiSignalIO`, :class:`BCI2000IO`, :class:`BiocamIO`,
+:class:`BlackrockIO`, :class:`IgorIO`, :class:`IntanIO`, :class:`KlustaKwikIO`, :class:`MaxwellIO`, :class:`MEArecIO`,
+:class:`NeoMatlabIO`, :class:`NeuralynxIO`, :class:`NixIO`, :class:`NWBIO`, :class:`OpenEphysIO`, :class:`PlexonIO`,
+:class:`Plexon2IO`, :class:`SpikeGLXIO`, and :class:`TdtIO`.
+
+Other changes
+-------------
+
+- RawIO classes have a method :meth:`parse_header` that extracts all the information from the formats
+  and therefore is usually a computational heavy process.
+  A flag ``is_header_parsed`` has now been added, to avoid re-extracting the header information
+  if it is already available. This can produce significant speed ups.
+
+- Type hints have been added to the :mod:`baseio`, :mod:`baserawio`, and :mod:`exampleio` modules,
+  with the aim of helping developers who wish to add a new IO module.
+
+- All of the code is now formatted with black_.
+
+- The continuous integration (CI) pipeline has been speeded up.
```

Comment:
Is this British English? We would say sped up, but I know British English says hotting up instead of heating up. So just wanted to double check.

Reply:
fixed. ("hotting up" is only really used metaphorically, to mean 'getting more interesting/tense/competitive', "heating up" would be used for cooking, the weather)